css-color-4/Overview.bs

```diff
@@ -2332,6 +2332,56 @@ Converting Between Uncalibrated CMYK and RGB-Based Colors</h3>
 		<li>fallback color must be set to the input color
 	</ul>
 
+
+<h2 id="css-system-colors">
+CSS System Colors</h2>
```

Comment:
As I mentioned above, I think it would likely be better if something in this section included the deprecated system colors in the `<system-color>` term so that other pieces of spec wouldn't miss half of them.

Reply:
Besides removing the definition of `<deprecated-system-color>`, would you mind clarifying what you'd like changed here? Move the deprecated list out of the appendix and into this section (though of course still noting they are deprecated)?

Reply:
@dbaron it looks like this PR is awaiting your clarification of what changes you requested.